site/en/docs/lighthouse/performance/mainthread-work-breakdown/index.md

```diff
@@ -13,7 +13,7 @@ is what turns your code into a web page that your users can interact with.
 By default, the [main thread](https://developer.mozilla.org/docs/Glossary/Main_thread)
 of the renderer process typically handles most code:
 it parses the HTML and builds the DOM, parses the CSS and applies the specified styles,
-and parses, evaluates, and executes the JavaScript.
+and parses, evaluates and executes the JavaScript.
```

Comment:
We use the [serial comma](https://developers.google.com/style/commas#serial-commas), so this change will be reverted.

Reply:
```suggestion
and parses, evaluates, and executes the JavaScript.
```